bazel/bazelrcs/docker.bazelrc

```diff
@@ -1 +1 @@
-common --config=docker
```

Comment:
This change is required because the `bazel query` in the test job otherwise uses an empty config which leads to an error.